subgraphs/balancer-v2/src/common/getters.ts

```diff
@@ -0,0 +1,285 @@
+import { Address, BigDecimal, BigInt, dataSource, ethereum } from "@graphprotocol/graph-ts";
+import {
+  Token,
+  DexAmmProtocol,
+  LiquidityPool,
+  LiquidityPoolFee,
+  FinancialsDailySnapshot,
+  UsageMetricsDailySnapshot,
+  Swap,
+  LiquidityPoolDailySnapshot,
+  LiquidityPoolHourlySnapshot,
+} from "../../generated/schema";
+import { fetchTokenSymbol, fetchTokenName, fetchTokenDecimals, scaleDown } from "./tokens";
+import {
+  BIGDECIMAL_ZERO,
+  Network,
+  VAULT_ADDRESS,
+  ProtocolType,
+  SECONDS_PER_DAY,
+  BIGINT_ZERO,
+  LiquidityPoolFeeType,
+  SECONDS_PER_HOUR,
+} from "./constants";
+import { WeightedPool as WeightedPoolTemplate } from "../../generated/templates";
+import { WeightedPool } from "../../generated/Vault/WeightedPool";
+import { ConvergentCurvePool } from "../../generated/Vault/ConvergentCurvePool";
+
+export function getOrCreateDex(): DexAmmProtocol {
+  let protocol = DexAmmProtocol.load(VAULT_ADDRESS.toHexString());
+  if (protocol === null) {
+    let network = dataSource.network();
+    if (network === "matic") {
```

Comment:
these can be changed to `network = dataSource.network().toUpper()`

Reply:
Btw with arbitrum we can not use the default name, we need to use `arbitrum_one` instead of `arbitrum-one`

Reply:
Can you update the logic here as well?